### PR TITLE
Update git to version 2.26.2

### LINKED ISSUE #3843: git: SSL certificate problem, unable to get local issuer certificate
after updating to Git 2.24.1-12, i keep getting for example:
```
fatal: unable to access 'https://github.com/SickChill/SickChill.git/': SSL certificate problem: unable to get local issuer certificate
```
This happens aswell when i manually pull a git, or clone a git.
I've tried reinstalling Git, but didn't work.

### PR DESCRIPTION
**Pull request by hgy59**
_Motivation:_  Initiated by the ca-certificate problem with the former package version.
_Linked issues:_  Closes #3843

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### Remarks
- configure system with DSM provided ca-certificates, that hopefully will be kept up to date by synology (and are more up-to-date than the ca-bundle of curl).
- add an option to build cross/curl without ca-bundle and use curl without ca-bundle for git. This fastens the build (download is done for each arch, without any caching), avoids outdated ca-certificates and reduces the package size

### CODE REVIEW

**Comment by publicarray** [comment]
LGTM

File: cross/curl/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PKG_NAME = curl
-PKG_VERS = 7.62.0
+PKG_VERS = 7.70.0
 PKG_EXT = tar.bz2
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = http://curl.haxx.se/download
```
Comment (publicarray) [inline cross/curl/Makefile:5]:
```suggestion
PKG_DIST_SITE = https://curl.haxx.se/download
```

File: cross/curl/Makefile [hunk 2]
```diff
@@ -12,19 +12,30 @@ COMMENT  = Command line tool and library for transferring data with URLs
 LICENSE  = http://curl.haxx.se/docs/copyright.html
```
Comment (publicarray) [inline cross/curl/Makefile:12]:
```suggestion
LICENSE  = https://curl.haxx.se/docs/copyright.html
```

**Comment by hgy59** [comment]
@publicarray I just merged your proposal for https in curl/Makefile.
And now the build actions are triggered with changes for all files that diff to master. This includes the files of the last commit on the master branch.
```
 .github/actions/build-push-pr.sh .github/actions/build-tag.sh .github/workflows/build.yml Makefile cross/comskip/Makefile cross/curl/Makefile cross/curl/PLIST cross/curl/digests cross/ffmpeg/Makefile cross/git/Makefile cross/git/PLIST cross/git/digests cross/git/src/gitconfig cross/gmmlib/Makefile cross/intel-media-driver/Makefile cross/intel-media-sdk/Makefile cross/iperf3/Makefile cross/iperf3/digests cross/lame/Makefile cross/libaom/Makefile cross/libvpx/Makefile cross/lirc-0.8.7/Makefile cross/lirc-0.8.7/PLIST cross/lirc-0.9.0/Makefile cross/lirc-0.9.0/PLIST cross/sabnzbd/Makefile cross/svt-av1/Makefile cross/svt-hevc/Makefile cross/tvheadend/Makefile cross/x264/Makefile cross/x265/Makefile mk/spksrc.dependency-tree.mk mk/spksrc.install-resources.mk mk/spksrc.spk.mk spk/git/Makefile spk/lirc/Makefile spk/synocli-disk/Makefile spk/synocli-file/Makefile spk/synocli-monitor/Makefile spk/synocli-net/Makefile
```

If this PR would have been rebased, the diff would be as small as possible.

Do you know a solution to get the diff for changes on the branch only?

**Pull request merged**